app/src/components/Project/Content/AllCardsContent.tsx

```diff
@@ -101,7 +101,7 @@ export default function AllCardsContent({
                 <RiArrowLeftSLine onClick={handleScrollForward} />
               </div>
             </div>
-            <div ref={scrollContainerRef} className='flex overflow-x-scroll'>
+            <div ref={scrollContainerRef} className='flex overflow-x-scroll snap-x snap-mandatory'>
               {filteredCards
                 .sort(({ createdAt: a }, { createdAt: b }) => b - a)
                 .map((card, index) => (
```

Comment:
please add the `className='snap-center'` to each rendered item. It should work =)

Reply:
sorry egueek i should remove 'snap-x snap-mandatory'

Reply:
no